apps/betterangels-backend/accounts/tests/test_permissions.py

```diff
@@ -150,3 +152,139 @@ def test_view_client_profiles_permission(self, user_label: str, should_succeed:
 
         expected_client_count = client_count if should_succeed else 0
         self.assertTrue(len(response["data"]["clientProfiles"]) == expected_client_count)
+
+
+@override_settings(DEFAULT_FILE_STORAGE="django.core.files.storage.InMemoryStorage")
+class ClientDocumentPermessionTestCase(ClientProfileGraphQLBaseTestCase):
```

Comment:
Mirroring attachment permission structure because we don't have document-specific permission requirements. I'm assuming that we'll want CMs from other orgs to be able to create and view client docs